sawtooth/cli.py

```diff
@@ -325,6 +418,13 @@ def main(prog_name=os.path.basename(sys.argv[0]), args=sys.argv[1:]):
         do_keygen(args)
     elif args.command == 'submit':
         do_submit(args)
+    elif args.command == 'block':
+        if args.url is not None:
```

Comment:
Let's put the implementation in a do_block() function similar to do_keygen/do_submit.  We will have a lot of these, making it uniform will be important.